content/ch-quantum-hardware/error-correction-repetition-code.ipynb

```diff
@@ -4536,7 +4713,7 @@
     }
    ],
    "source": [
-    "qobj = assemble(qc_init+qc)\n",
+    "qobj = assemble(qc.compose(qc_init, front=True))\n",
```

Comment:
I'm not too familiar with `compose`, is there a reason for using `front=True` instead of doing this the other way round? (`qc_init.compose(qc)`)

Reply:
@frankharkins Sorry for the late response. As mentioned in the [docs](https://qiskit.org/documentation/stubs/qiskit.circuit.QuantumCircuit.html#qiskit.circuit.QuantumCircuit.compose), `qc` has to be "wider" than `qc_init`. Not quite sure what `front=True` actually does (it's mentioned that it performs front decomposition, which I'm not aware of) but on testing the codes, using `front=True` exactly replicates the deprecated code used before.

Reply:
Ok I see. In that case, LGTM.